packages/renderer/src/lib/preferences/PreferencesCliTool.svelte

```diff
@@ -46,8 +53,38 @@ async function update(cliTool: CliToolInfo) {
   } catch (e) {
     showError = true;
   } finally {
-    cliToolStatus.inProgress = false;
-    cliToolStatus = cliToolStatus;
+    cliToolUpdateStatus.inProgress = false;
+    cliToolUpdateStatus = cliToolUpdateStatus;
+  }
+}
+
+async function install(cliTool: CliToolInfo) {
+  // user has to select the version to install
+  let versionToInstall;
+  try {
+    versionToInstall = await window.selectCliToolVersionToInstall(cliTool.id);
+  } catch (e) {
+    // do nothing
+    console.log(e);
```

Comment:
```suggestion
    console.error(e);
```

Reply:
error shouldn't be displayed to the end user or thrown ?

like not selecting a version and having an error in the select looks different usecase ?

Reply:
i took advantage of the error message already in the component to show it

Reply:
thanks